bin/deploy.js

```diff
@@ -0,0 +1,18 @@
+const ghPages = require('gh-pages');
+
+const { print, runFromTheme, getGithubUrl } = require('./utils');
+
+const deploy = async () => {
+  runFromTheme(); // move to the directory of React Static
+  const githubUrl = getGithubUrl();
+
+  await ghPages.publish('dist', {
+    message: 'Deployed by landr 🏠',
+    branch: 'gh-pages',
+    repo: githubUrl,
+  });
+
+  print.success('Website successfully build and deployed 🏠`');
```

Comment:
NIT: there are some typos here
```suggestion
  print.success('Website successfully build and deployed 🏠');
```